Content.Shared/Body/Prototypes/BodyPrototypeSerializer.cs

```diff
@@ -14,18 +14,10 @@ namespace Content.Shared.Body.Prototypes;
 [TypeSerializer]
 public sealed class BodyPrototypeSerializer : ITypeReader<BodyPrototype, MappingDataNode>
 {
-    private (ValidationNode Node, List<string> Connections) ValidateSlot(ISerializationManager serializationManager, MappingDataNode slot, string slotId, IDependencyCollection dependencies)
+    private (ValidationNode Node, List<string> Connections) ValidateSlot( MappingDataNode slot, IDependencyCollection dependencies)
     {
         var nodes = new List<ValidationNode>();
         var prototypes = dependencies.Resolve<IPrototypeManager>();
-        if (!slot.TryGet("part", out ValueDataNode? part))
-        {
-            nodes.Add(new ErrorNode(slot, $"No part value data node found in root slot {slotId}"));
-        }
-        else if (!prototypes.HasIndex<EntityPrototype>(part.Value))
-        {
-            nodes.Add(new ErrorNode(slot, $"No entity prototype found with id {part.Value} for root slot {slotId}"));
-        }
```

Comment:
This check is still needed to validate that part ids exist as entities, either that or a prototype id serializer should be used for the `Part` property as its custom type serializer.